mitodl/mit-open
# Merge `main` into `nextjs`

### What are the relevant tickets?

Relates to https://github.com/mitodl/hq/issues/5395

### Description (What does it do?)


Updates the Next.js branch with changes in `main`.


### How can this be tested?


Ensure that the branch runs locally and everything looks in order and recent changes in `main` have been applied.

Special attention to changes to the Topic Detail Page for https://github.com/mitodl/mit-open/pull/1646 as this was the biggest change to the frontend code, plus the "Content File Score Weight Adjustment" admin search control for https://github.com/mitodl/mit-open/pull/1667.
